src/winter_onboarding_2021/fleet_management_service/routes.clj

```diff
@@ -19,9 +19,10 @@
         ["cabs" {"" {:get (wrap-layout handlers/get-cabs)
                      :post handlers/create}
                  "/new" {:get (wrap-layout handlers/new)}
-                 ["/" :id] {"/edit" {:get (wrap-layout handlers/update-cab-view)}
-                            :get (wrap-layout handlers/view-cab)
-                            :post handlers/update-cab}}]
+                 ["/" :id] {:get (wrap-layout handlers/view-cab)
+                            :post handlers/update-cab
+                            "/edit" {:get (wrap-layout handlers/update-cab-view)}
+                            "/delete" {:get handlers/delete}}}]
```

Comment:
This should not be a get request. You can make it a POST request and call the API using a form submit action. 

Technically, in an API world this should be a DELETE request.

Reply:
Making this a GET would also have severe security implications.